dependencies.gradle

```diff
@@ -29,7 +29,7 @@ def jjwt = "0.11.5"
 def vanniktechEmoji = "0.15.0"
 
 // Testing
-def mockk = "1.12.4"
+def mockk = "1.12.3"
```

Comment:
Is there a reason to downgrade here? If yes, please add a comment above this line, else Dependabot will change this line again and we will have some trouble.